examples/basic_example.py

```diff
@@ -40,7 +40,7 @@ def future_loading(t, x = None):
     if isinstance(filt, state_estimators.UnscentedKalmanFilter):
         samples = filt.x.sample(20)
     else: # Particle Filter
-        samples = filt.x.raw_samples()
+        samples = filt.x
```

Comment:
With the refactor I make raw_samples an depreciated call, so I'm updating the example to not use it